src/mame/ussr/bk.cpp

```diff
@@ -55,10 +54,73 @@ void bk_state::bk0010fd_mem(address_map &map)
 	map(0120000, 0157777).ram();
 }
 
+void bk_state::bk0011_mem(address_map &map)
+{
+	map(0000000, 0177777).rw(FUNC(bk_state::trap_r), FUNC(bk_state::trap_w));
+
+	map(0000000, 0037777).ram().share(m_ram[0]);
+
+	map(0040000, 0077777).view(m_view1);
```

Comment:
You can have for loops in memory maps, it's normal C++ code after all, that may make things a little less copypasty.

Reply:
Yes, but in this case key order doesn't match (6, 0, ..., 5 -> 0, 1, ..., 7)